Unit-5/Overview.md

```diff
@@ -1,26 +1,28 @@
-### Overview:
+### Overview
 
-In unit 4 we explored the introduction to research reports we saw that the research problem establishes why a particular research study is worth doing, the review of literature establishes what is already known about the problem, and the research purpose establishes what the research specifically intends to do.  In unit 5 we want to extend this to the next major section of a research report:  The methods and results.  In this unit, we will see that the methods section indicates the overall plan for how the research was conducted, how the data was collected, and what the analysis of the data tells us.  These may be termed research design, method, and analysis and results, respectively.  Because of the great differences in quantitative and qualitative methods, we will break this discussion into two units.  This unit will focus on the methods and results of quantitative research reports, while unit 6 will focus on those of qualitative research reports.
+In unit 4, as we explored the introduction to research reports, we saw that the research problem establishes why a particular research study is worth doing, the review of literature establishes what is already known about the problem, and the research purpose establishes what the research specifically intends to do.  In unit 5 we want to extend this to the next major section of a research report: The methods and results. In this unit, we will see that the methods section indicates the overall plan for how the research was conducted, how the data was collected, and what the analysis of the data tells us. hese may be termed research design, method, and analysis and results, respectively.  Because of the great differences in quantitative and qualitative methods, we will break this discussion into two units.  This unit will focus on the methods and results of quantitative research reports, while unit 6 will focus on those of qualitative research reports.
```

Comment:
It looks like there is a "T" missing from 4th sentence in this section.  Did that get deleted by accident?

Reply:
From the beginning of the sentence from the word "These"

Reply:
Sorted.